### ISSUE TITLE
Adblock hide social media class

### ISSUE THREAD

**Issue by Embraser01** [Lednerb/bilberry-hugo-theme#95]
Hi,

I discovered (after some time :cry:) that adblock was setting a `display: none !important` on a lot of element. 

The `social-media` is one of them. 

Also, I found this gist: https://gist.github.com/spyesx/42fe84c0ef757d1c38a4
which list all css class name we should stop using.

Is-it possible to rename every css class concerned by this thing?

**Comment by Lednerb** [comment]
Hi and thank you for the hint.

I didn't thought about that but in the first moment now I think this is an adblock related problem...

Sure adblock is widely used and so it will affect the users of this theme...

Maybe we can implement a hint to better use uBlock origin that does imho a better job at all...

------

On the other side, maybe this should be the aspected behavior? If I'm using adblock, maybe I don't want to see the Facebook / Twitter etc. stuff? Maybe I would be annoyed to see those links?

------

But yeah, we can check if we can rename the classes but have to check if using the font-awesome icons is still possible.

**Comment by Embraser01** [comment]
> On the other side, maybe this should be the aspected behavior? If I'm using adblock, maybe I don't want to see the Facebook / Twitter etc. stuff? Maybe I would be annoyed to see those links?

For me it is intended to hide sharing links, not to hide links to specific profiles. If I go to a website of someone, I expect to see his social networks (I only use Github & LinkedIn)...

I don't really like the idea of having any adblocker related message on my website which does not contain any ad. 

As I see, Adblock will hide some font awesome icon, but not all of them:
```
.fa-facebook,
.fa-facebook-square,
.fa-google-plus,
.fa-share-alt,
.fa-twitter,
.fa-youtube-play,
````

I think we could rename the `social-media` class anyway to display the block (and be easier to debug 😉), then find a solution for each icon?

**Comment by Lednerb** [comment]
> I don't really like the idea of having any adblocker related message on my website which does not contain any ad.

I absolutely confirm this. This kind of messages are annoying, and personally, I've just enabled a filter list for uBlock origin to block the dim-witted Cookie messages that ask for consent as part of the EU laws.

-------------

> I think we could rename the social-media class anyway to display the block (and be easier to debug :wink:), then find a solution for each icon?

Sure we should do this, `social-media` is even not correct at all...
We can also add classes for the blocked ones, maybe `.adblock-proof-fa-facebook`, etc. for the blocked ones with the same content as the original `.fa-*` classes.

By adding this we have a good readability and also unblock the icons.

**Comment by Lednerb** [comment]
Implemented a fix. Will be released in Version 1.8.0
